bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -2881,5 +2881,18 @@ Mapping method from excess ice input stream data to the model resolution
     none     = no interpolation
 </entry>
 
+<!-- ========================================================================================  -->
+<!-- Namelist items controlling tillage                                                        -->
+<!-- ========================================================================================  -->
+
+<entry id="tillage_mode" type="char*8" category="physics"
+       group="tillage_inparm" valid_values="off,low,high">
+Whether to till crop soil, and if so, with what intensity.
+</entry>
+
+<entry id="use_original_tillage" type="logical" category="physics"
+       group="tillage_inparm" valid_values="" value=".false.">
+Toggle to use original tillage logic
```

Comment:
Unfortunately I can't tell where it was introduced. It's present in Mike's code, but the git history isn't helpful; there's no link to Sam L.'s code in the publication you linked. Ultimately I think it doesn't matter where it was introduced, though, because Sam L.'s paper only looked at sites in the US Midwest, where I don't expect seasons to ever cross into a new calendar year.

I've updated the description with commit b71fa23:
> Toggle to use original tillage logic, with bug for seasons crossing into a new calendar year

What do you think?